cmd/aro/operator.go

```diff
@@ -194,6 +195,14 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 		}
 	}
 
+	if role == pkgoperator.RoleWorker {
```

Comment:
I believe we could run this controller on the master operator alongside our other controllers - the purpose of the worker operator is mostly just to run network checks within the worker subnet, so I am not sure we need to run the controller there.

Reply:
I see, I had thought this controller needed to only run on workers as we don't want the MHC to operate on masters